src/cascadia/TerminalSettingsEditor/ViewModelHelpers.idl.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
`.idl.h`? what the heck

Reply:
YOU CAN DEFINE MACROS FOR THE IDL

Reply:
WHAT

Reply:
YEP! I'm using `.idl.h` as a signal/suggestion, not a contract.